charts/nextcloud/files/notify_push.sh.tpl

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+/var/www/html/occ app:install notify_push
+/var/www/html/occ config:app:set notify_push base_endpoint --value="https://{{ .Values.nextcloud.host }}{{ .Values.notifyPush.ingress.path }}"
```

Comment:
Hardcoded to https? This seems wrong to me.
Also accessing it through the external host shouldn't be necessary and wouldn't work if the port is different.

Reply:
Or alternatively the hostname in the container should be bent to localhost similar to https://github.com/NixOS/nixpkgs/blob/a23b0bdc68f56700d03aec9598eeeae6975f0ade/nixos/modules/services/web-apps/nextcloud-notify_push.nix#L138-L141